src/main/resources/application.properties

```diff
@@ -53,68 +52,53 @@ spring.data.mongodb.database: eiffel_intelligence
 # if these properties are empty, remove or comment them
 #spring.data.mongodb.username:
 #spring.data.mongodb.password:
-
 # session expiration
 server.session-timeout: 1200
+# Possible to configure a name for the database sessions collection
 sessions.collection.name: sessions
 
-# name of the collection where aggregated objects are stored
+# name of the database collection where aggregated objects are stored
 aggregated.collection.name: aggregated_objects
-# time to live value in seconds for aggregated objects.
-# this needs to be set if using all_events_rules to avoid a copy of
-# event repository. Recommended setting is 10 minutes
+# time to live value (seconds) for aggregated objects.
+# this needs to be set if using the example rule set 'AllEventsRules' to avoid the growth
+# of storage too big. Recommended setting is 10 minutes
 aggregated.collection.ttlValue:
-# the name of the collection where to store the maping between 
-# aggregated objects and id of the events that contributed to them
+# the name of the database collection where to store the mapping between
+# aggregated objects and id of the Eiffel events that contributed to those
 event_object_map.collection.name: event_object_map
-# name of the collection where unprocessed events are stored
+# name of the database collection where unprocessed Eiffel events are stored
```

Comment:
Explain that waitlist keeps a cashe of eiffel events stored in MongoDB waiting for processing.